master/buildbot/test/unit/test_process_buildrequest.py

```diff
@@ -64,6 +64,7 @@ def test_collapseRequests_no_other_request(self):
         def collapseRequests_fn(builder, brdict1, brdict2):
             # Allow all requests
             self.fail("Should never be called")
+            # XXX(sa2ajj): Dead in the water?
```

Comment:
[The docs](https://docs.python.org/2/library/unittest.html#unittest.TestCase.fail) don't say that fail doesn't return..